[{"exception": "error", "inner": [{"provider": {"package": "java.lang", "class": "Throwable", "method": "<init>", "filename": "Throwable", "line": ""}, "caller": {"package": "org.eclipse.jface.text", "class": "Assert$AssertionFailedException", "method": "<init>", "filename": "Assert", "line": "54"}}], "outer_pre": [], "outer_suf": [], "all": [{"provider": {"package": "java.lang", "class": "Throwable", "method": "<init>", "filename": "Throwable", "line": ""}, "caller": {"package": "org.eclipse.jface.text", "class": "Assert$AssertionFailedException", "method": "<init>", "filename": "Assert", "line": "54"}}]}]